peg-macros/translate.rs

```diff
@@ -232,18 +239,17 @@ fn compile_rule(context: &Context, rule: &Rule) -> TokenStream {
     let body = compile_expr(&context, &rule.expr, rule.ret_type.is_some());
 
     let wrapped_body = if cfg!(feature = "trace") {
-        let str_rule_name = rule.name.to_string();
         quote_spanned! { span => {
             let loc = ::peg::Parse::position_repr(__input, __pos);
-            println!("[PEG_TRACE] Attempting to match rule `{}` at {}", #str_rule_name, loc);
+            tracing::trace!(loc = %loc, "Attempting");
             let __peg_result: ::peg::RuleResult<#ret_ty> = {#body};
             match __peg_result {
                 ::peg::RuleResult::Matched(epos, _) => {
                     let eloc = ::peg::Parse::position_repr(__input, epos);
-                    println!("[PEG_TRACE] Matched rule `{}` at {} to {}", #str_rule_name, loc, eloc);
+                    tracing::trace!(eloc = %eloc, "Matched");
```

Comment:
`loc` argument is loosed intentionally?

Reply:
It was implied by the `instrument` macro, that created a span of level to show that we have entered this location now. I deemed it to be a redundant information so I do not add it back

Reply:
It just strange that you omit `loc` here, and at the same time keep it in line 237 (the values of `loc` was the same in those two places). Oversight?